static/app/components/gridEditable/styles.tsx

```diff
@@ -57,7 +57,7 @@ export const Body = styled(
     showVerticalScrollbar?: boolean;
   }) => (
     <Panel {...props}>
-      <PanelBody>{children}</PanelBody>
+      <PanelBody style={{height: '100%'}}>{children}</PanelBody>
```

Comment:
I would prefer if you restyled PanelBody, so that we don't have multiple different ways of defining CSS (it makes it harder to track values down if we need to look in multiple places)